README.md

```diff
@@ -1256,6 +1256,674 @@ The type of cost
+
+### The track_markers_with_torque_actuators.py file
+This example is a trivial box that must superimpose one of its corner to a marker at the beginning of the movement
+and superimpose the same corner to a different marker at the end. It is a clone of
+'getting_started/custom_constraint.py' It is designed to show how to use the `TORQUE_ACTIVATIONS_DRIVEN` which limits
+the torque to [-1; 1]. This is useful when the maximal torque are not constant. Please note that this dynamic then
+to not converge when it is used on more complicated model. A solution that defines non-constant constraints seems a
+better idea. An example of which can be found with the bioptim paper.
+
+Let's take a look at the structure of the code. First, tau_min, tau_max and tau_init are respectively initialized 
+to -1, 1 and 0 if the integer `actuator_type` (which is a parameter of the `prepare_ocp` function) equals to 1. 
+In this particular case, the dynamics function used is `DynamicsFcn.TORQUE_ACTIVATIONS_DRIVEN`. 
+
+### The trampo_quaternions.py file
+
+This example uses a representation of a human body by a trunk_leg segment and two arms.
+It is designed to show how to use a model that has quaternions in their degrees of freedom.
+
+## Track
+In this section, you will find the description of two tracking examples. 
+
+### The track_marker_on_segment.py file
+This example is a trivial example where a stick must keep a corner of a box in line for the whole duration of the
+movement. The initial and final position of the box are dictated, the rest is fully optimized. It is designed
+to show how one can use the tracking function to track a marker with a body segment.
+
+In this case, we use the `ConstraintFcn.TRACK_MARKER_WITH_SEGMENT_AXIS` constraint function, as shown below in the 
+definition of the constraints of the problem:
+
+```python
+constraints = ConstraintList()
+constraints.add(
+ConstraintFcn.TRACK_MARKER_WITH_SEGMENT_AXIS, node=Node.ALL, marker_idx=1, segment_idx=2, axis=Axis.X
+)
+```
+
+Here, we minimize the distance between the marker with index 1 ans the x axis of the segment with index 2. We align 
+the axis toward the marker. 
+
+### The track_segment_on_rt.py file
+This example is a trivial example where a stick must keep its coordinate system of axes aligned with the one
+from a box during the whole duration of the movement. The initial and final position of the box are dictated,
+the rest is fully optimized. It is designed to show how one can use the tracking RT function to track
+any RT (for instance Inertial Measurement Unit [IMU]) with a body segment.
+
+To implement this tracking task, we use the `ConstraintFcn.TRACK_SEGMENT_WITH_CUSTOM_RT` constraint function, which 
+minimizes the distance between a segment and an RT. The extra parameters `segment_idx: int` and `rt_idx: int` must be 
+passed to the Objective constructor.
+
+## Moving estimation horizon
+In this section, we perform mhe on the pendulum example.
+
+### The mhe.py file
```

Comment:
This example is way too long

Reply:
As I mentionned in the previous conversation, I completely changed the structure of this example.

Reply:
Done

Reply:
gg